concordium-node/src/configuration.rs

```diff
@@ -74,6 +72,7 @@ pub const SOFT_BAN_DURATION_SECS: u64 = 300;
 pub const MAX_PEER_NETWORKS: usize = 20;
 /// Database subdirectory name
 pub const DATABASE_SUB_DIRECTORY_NAME: &str = "database-v4";
+const HOUSE_KEEPING_INTERVALS_PER_NORMAL_KEEP_ALIVE: u8 = 3;
```

Comment:
Please add some documentation of this constant and the rationale for it.

Reply:
Tried doing this. Would like feedback on whether it suffices!

Reply:
What does the `Normal` refer to?
How about renaming it to simply just `KEEP_ALIVE_FACTOR`.

And then use // for explaining it.
```
In order to avoid prematurely connection drops, then the 'KEEP_ALIVE_FACTOR' must be at least 3 times greater than the 'housekeeping_interval'.
```

Reply:
What does the Normal refer to?
How about renaming it to simply just KEEP_ALIVE_FACTOR.

And then use // for explaining it.

_In order to avoid prematurely connection drops, the 'KEEP_ALIVE_FACTOR' must be at least 3 times greater than 'housekeeping_interval'._

Reply:
The `normal` is there, as there is is also a `MAX_BOOTSTRAP_KEEP_ALIVE`, however I didn't like the name to begin with.. I'll apply yours.